waiter/config-full.edn

```diff
@@ -136,8 +136,8 @@
                   ;; When SSL is enabled, the password to the keystore
                   :key-password "keystore-password"
 
-                  ;; The maximum number of threads to use (default 250)
-                  :max-threads 250
+                  ;; The maximum number of threads to use (default 200)
+                  :max-threads 200
```

Comment:
Why did this change?

Reply:
I reduced this since we no longer perform the InputStream reads on the Jetty threads. Having lower number of threads (though net we are ending up increasing it by a 100 due to the input stream reader thread pool) reduces the pressure on the JVM and the heap.